.gitignore

```diff
@@ -79,3 +79,4 @@ vendor/
 keystore.jks
 truststore.jks
 notes.txt
+quaycreds.sh
```

Comment:
These seem like arbitrary files

Reply:
local files needed for testing purposes. does it matter?